ingestr/src/sources.py

```diff
@@ -131,6 +131,11 @@ def dlt_source(self, uri: str, table: str, **kwargs):
 
         if uri.startswith("mysql://"):
             uri = uri.replace("mysql://", "mysql+pymysql://")
+        
+        if uri.startswith("clickhouse://"):
+            uri = uri.replace("clickhouse://", "clickhouse+native://")
+            if "secure=" not in uri:
+              uri += "?secure=1"
```

Comment:
This won't work in all cases. For example if the user specifies the uri as 

`clickhouse://user:pass@host?http=port`

It'll add another `?` at the end, which isn't a valid URI